tests/conftest.py

```diff
@@ -41,11 +50,13 @@ def pages():
 
 
 @pytest.fixture(autouse=True)
+# Performs setup and tear down
 def create_driver(prep_properties, request):
     global driver
     browser = request.config.option.browser
     config_reader = prep_properties
     base_url = config_reader.config_section_dict("Base Url")["base_url"]
+    write_to_allure_env_file({"browser": browser, "base_url": base_url})
```

Comment:
we want it one time per session not for each test
use:
https://stackoverflow.com/a/32573090/4515129